src/login/logind-dbus.c

```diff
@@ -3213,6 +3307,8 @@ const sd_bus_vtable manager_vtable[] = {
         SD_BUS_METHOD("ScheduleShutdown", "st", NULL, method_schedule_shutdown, SD_BUS_VTABLE_UNPRIVILEGED),
         SD_BUS_METHOD("CancelScheduledShutdown", NULL, "b", method_cancel_scheduled_shutdown, SD_BUS_VTABLE_UNPRIVILEGED),
         SD_BUS_METHOD("Inhibit", "ssss", "h", method_inhibit, SD_BUS_VTABLE_UNPRIVILEGED),
+        SD_BUS_METHOD("CanRebootParam", NULL, "s", method_can_reboot_param, SD_BUS_VTABLE_UNPRIVILEGED),
+        SD_BUS_METHOD("SetRebootParam", "s", NULL, method_set_reboot_param, SD_BUS_VTABLE_UNPRIVILEGED),
```

Comment:
one nitpick: Let's avoid needless abbreviations. let's please call this RebootParameter instead of RebootParam in public APIs

Reply:
Ok. I also adjusted other reboot param references to full reboot parameter.